.babelrc.js

```diff
@@ -56,7 +59,7 @@ module.exports = {
   plugins: plugins,
   overrides: [
     {
-      test: /.*logger.*/,
+      test: /.*logger\..*/,
```

Comment:
Whats this change doing?

Reply:
This change is used to fix an error for when testHelpers.js is removed later on.